Assets/Standard Assets/Characters/FirstPersonCharacter/Scripts/FirstPersonController.cs

```diff
@@ -19,6 +19,7 @@ public class FirstPersonController : MonoBehaviour
         [NonSerialized] public float keyLocationSpeedMod = 1;
         [SerializeField] public bool isPlayerKeyLocXPositive;
         [SerializeField] public bool isPlayerKeyLocZPositive;
+        [SerializeField] public bool isPlayerInKeyLocZone = false;
```

Comment:
No need to have a combination of both `[SerializeField]` and `public` here. I think just leave them public if you need to access them somewhere outside the class